.pre-commit-config.yaml

```diff
@@ -29,7 +29,7 @@ repos:
       - id: nbstripout
         types: [ "jupyter" ]
         args: [ "--drop-empty-cells",
-                "--extra-keys 'metadata.language_info.version cell.metadata.jp-MarkdownHeadingCollapsed cell.metadata.pycharm'" ]
+                "--extra-keys=metadata.language_info.version cell.metadata.jp-MarkdownHeadingCollapsed cell.metadata.pycharm" ]
```

Comment:
Good catch!

Does this still use all 3 keys? I would expect that it needs quotation marks around them.